packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -17,6 +17,7 @@ export function accordionItemTemplate<T extends FASTAccordionItem>(
             role="heading"
             aria-level="${x => x.headinglevel}"
         >
+            ${startSlotTemplate(options)}
```

Comment:
We might consider documenting that if this is interactive it'll break a11y with this change. I _think_ the reason that we had both start/end *after* the semantic button might have been related to that.

Reply:
Good catch. Focus was first going to the `start` element, then the `button`, then the `end` element. This is a good edge case as the aria guidelines don't address interactive contents in the header. I started by setting the `tabindex` on the button, which worked well for keyboard navigation, but it still read the start content before the button label. I put the start slot back after the button.

This is unfortunate since the styling to make the button appear to be the full size of the header is external to the template, but it's probably the safest solution.